src/index.html

```diff
@@ -430,6 +430,7 @@ <h3>Latest blog</h3>
       </div>
       <div class="row">
         <div class="col-4">
+         <a href="#" class="not-underline">
```

Comment:
`a` tag indented by 1 space. Also, the content of this `a` is indented by 1 space.